tests/regression/roles/sonic_vlans/defaults/main.yml

```
@@ -38,54 +43,62 @@ tests:
       - vlan_id: 500
         description: "modified vlan500 description"
       - vlan_id: 501
```

It would provide better coverage to add an explicit deletion case for 'autostate' == true. I realize that this should be a functional no-op, but some verification that the correct behavior occurs for this case would be good. For full vlan deletion, the case can be moved from vlan 501 (now set to 'autostate' = true, so needed in the new case I'm requesting here) to vlan 502.

```suggestion
      - vlan_id: 500
        description: "modified vlan500 description"
      - vlan_id: 501
        autostate: true
      - vlan_id: 502
```